docs/custom-base-image.md

```diff
@@ -12,10 +12,17 @@ You should always use a specific version which can be found at [docker hub](http
 Use this template for using a custom base image with our default user named `user` and userid `1000`.
 
 ```dockerfile
+# This default may be swapped for any compatible base image
+ARG BASE_IMAGE=amd64/ubuntu:20.04
+
 # This buildpack is used for tool intallation and user/directory setup
 FROM containerbase/buildpack AS buildpack
 
-FROM amd64/ubuntu:focal as base
+FROM ${BASE_IMAGE} as base
+
+ARG USER_NAME=ubuntu
```

Comment:
Did you read the description above the sample? It was intended to not override User here.

Reply:
I had the build fail without it. Will try again

Reply:
If it won't work there's a bug